src/applications/personalization/dashboard-2/tests/unit/healthcare-spinner.unit.spec.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderInReduxProvider } from '~/platform/testing/unit/react-testing-library-helpers';
+import reducers from '~/applications/personalization/dashboard/reducers';
+import HealthCare from '~/applications/personalization/dashboard-2/components/health-care/HealthCare';
+
+describe('HealthCare component', () => {
+  let view;
+  let initialState;
+
+  context('when appointments and messaging data are still loading', () => {
+    it('should only show a loading spinner', async () => {
+      initialState = {
+        user: {
+          profile: {
+            services: ['messaging'],
+          },
+        },
+        health: {
+          appointments: {
+            fetching: true,
```

Comment:
I'm not 100% sure what changes could be made here, but if you stop using the `dataLoadingDisabled` flag as suggested below, I don't think you'll need to set `fetching: true` or `loading: true` on line 27. The component will mount, start fetching data, and set the redux state to the "loading" state.

Reply:
You were correct! Removed, and the test still passes. 🏆